### PR TITLE
Update PPP (and for consistency), and updated UG and DG

### PR DESCRIPTION
**Pull request by OngDeZhi**
Update:
1. PPP (rephrasing and for consistency).
2. UG (diagrams, fix phrasing and formatting).
3. DG (fix phrasing).

### CODE REVIEW

**Review by michaeldinata — APPROVED** [review]
LGTM! :)

File: docs/UserGuide.md [hunk 1]
```diff
@@ -289,7 +289,7 @@ Genre: [Action, Adventure, Drama, Sci-Fi]
 ### 3.6 Workspace management
 The workspace management feature provides translators the flexibility of organising their application data like watchlist and bookmark easily.
 
-The following subsections below describes the various workspace-relation operations.
+The following subsections below describes the various workspace-related operations.
```
Comment (ChanJianHao) [inline docs/UserGuide.md:292]: Thanks for spotting this :P

File: docs/UserGuide.md [hunk 2]
```diff
@@ -669,17 +670,16 @@ Sayonara <NAME>!
...
 
 ### 3.10 Saving and loading data
-
-User, workspace(s), watchlist(s), and bookmark(s) data will be **saved automatically** when they are created or modified, and will be **loaded automatically** when **AniChan** is launched. 
+User, workspace(s), watchlist(s), and bookmark(s) data will be **saved automatically** when they are created or modified, and they will also be **loaded automatically** when **AniChan** is launched. 
 
 In the folder where **AniChan** is launched, there will be a `data` folder which would contain these data:
 *   User data will be stored in `data/user.txt`.
-*   Watchlist(s) data will be stored in `data/<WORKSPACE-NAME>/watchlist.txt`, e.g. if "AniTranslator" is the name of your workspace, then the watchlist data can be found in `data/AniTranslator/watchlist.txt`.
+*   Watchlist(s) data will be stored in `data/<WORKSPACE-NAME>/watchlist.txt`. For example, if "AniTranslator" is the name of your workspace, then the watchlist data can be found in `data/AniTranslator/watchlist.txt`.
```
Comment (ChanJianHao) [inline docs/UserGuide.md:677]: Good work adding this!

File: docs/team/chanjianhao.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Chan Jian Hao - Project Portfolio Page
 
-## Project: AniChan
+## Overview
```
Comment (ChanJianHao) [inline docs/team/chanjianhao.md:3]: Thanks!

**Review by n3wsoldier — APPROVED** [review]
LGTM :)

File: docs/UserGuide.md [hunk 4]
```diff
@@ -129,12 +129,12 @@ This command allows you to estimate the time needed to translate a script. It es
...
 Format: `estimate <SCRIPT_FILE_NAME> [-wph WORDS_PER_HOUR]`
 *   It only accepts one `.txt` file (other file extensions will be supported in a future version).
-*   If the parameter `-wph` is not specified, the average translator's speed will be used to generate three estimations for you to consider.
+*   If the parameter `-wph` is not specified, the average translator's speed will be used to generate three timings for you to consider.
 
 > :bulb: Remember to specify the file extension. For example, `script.txt` and not `script`.
 
 > :bulb: Put the script in the current workspace folder. If the current workspace is "Default", then from the folder containing `AniChan.jar`, put it in the subfolder named "Default" of the "data" folder (i.e. `data/Default/script.txt`) as shown in the diagram below. 
-> <br/><br/> ![Estimate Folder Structure Diagram](images/Estimate-Folder-Structure-Diagram.png)
+> <br/><br/> ![Estimate Folder Structure Diagram](images/Estimate-Folder-Structure-Diagram.png) <br/> *Figure 3: Example of Folder Structure for Script Files*
```
Comment (ChanJianHao) [inline docs/UserGuide.md:137]: Amazing!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -110,7 +107,10 @@ E.g. `estimate <SCRIPT_NAME> [-wph WORDS_PER_HOUR]` can be used as `estimate scr
 *   Order of parameters (`-n`, `-l`, etc.) are not important, but they have to be defined after the non-parameter fields. <br/>
 E.g. Both `browse -s name -p 2` and `browse -p 2 -s name` gives the same result, but in the case of `estimate script.txt -wph 300` and `estimate -wph 300 script.txt`, the latter will not work because "script.txt" is not a parameter.
 
-> :bulb: This is what the input prompt means: **WORKSPACE-NAME (WATCHLIST-NAME) #>**
+<br/>
+
+> :bulb: AniChan's Input Prompt: 
+> <br/> <br/> ![AniChan's Input Prompt](images/AniChan-Input-Prompt.png) <br/> *Figure 2: AniChan's Input Prompt*
```
Comment (ChanJianHao) [inline docs/UserGuide.md:113]: Looks great! But could the image be resized smaller?

Comment (OngDeZhi) [reply docs/UserGuide.md:113]: Noted, I have resized it already. We could adjust it again if it is still too big or too small :)

**Review by ChanJianHao — APPROVED** [review]

**Pull request merged**